service/raft/raft_sys_table_storage.cc

```diff
@@ -303,6 +304,9 @@ future<> raft_sys_table_storage::execute_with_linearization_point(std::function<
 
 future<> raft_sys_table_storage::bootstrap(raft::configuration initial_configuation, bool nontrivial_snapshot) {
     auto init_index = nontrivial_snapshot ? raft::index_t{1} : raft::index_t{0};
+    utils::get_local_injector().inject("raft_sys_table_storage::bootstrap/init_index_0", [&init_index] {
```

Comment:
Do we need to backport this if it is not used?

Reply:
No, I'll remove